contracts/ABonding.sol

```diff
@@ -126,6 +128,7 @@ abstract contract ABonding is IBasicBonding, Ownable, ERC721Holder {
     function bondAmountOut(uint amountIn) public view returns (uint amountOut) {
         (int256 gtonPrice, uint gtonDecimals) = tokenPriceAndDecimals(gtonAggregator);
         (int256 tokenPrice, uint tokenDecimals) = tokenPriceAndDecimals(tokenAggregator);
+        uint tokenInUSD = amountIn * uint(tokenPrice) / tokenDecimals;
```

Comment:
Where do we use tokenInUSD?

Reply:
That is old code merge lines that got appended here. I will delete it